opencensus/trace/internal/span_impl.h

```diff
@@ -106,6 +106,7 @@ class SpanImpl final {
   friend class ::opencensus::trace::exporter::SpanExporterImpl;
   friend class ::opencensus::trace::SpanTestPeer;
 
+  void ExportSpanForTesting(const std::shared_ptr<SpanImpl>& span);
```

Comment:
1. Let's submit this as-is to unbreak Travis.

2. This seems a strange place to put this function. It should be static since it's not using any members. And it's only used to add a shared_ptr wrapper around `*this`, which it doesn't have access to.